scripts/application_to_provider.py

```diff
@@ -36,6 +40,26 @@
 If a non-public application has or more members without identities on the
 destination provider, script will exit with error unless
 --ignore_missing_members is set.
+
+The iRODS transfer feature was developed for CyVerse Atmosphere(0); may be of
+limited use elsewhere. In order to use it:
+- Source and destination providers must use the iRODS storage backend for
+  OpenStack Glance (https://github.com/cyverse/glance-irods)
+- Src. and dst. providers must store images in the same iRODS zone
+- --source-provider-id, --irods-conn, --irods-src-coll, and --irods-dst-coll
+  must all be defined
```

Comment:
Would seem that only _one_ of the two collections would be required if someone wanted to go `iRODS -> Glance` or `Glance -> iRODS`?

Just curious how far we are away from Many-to-many relationship 😄